### DISCUSSION TITLE
Parameter pack could not be expanded within '...'

### DISCUSSION THREAD

**Discussion by MengWWWWW** [eProsima/Fast-DDS#4666] [Q&A]
"I encountered an error while installing Fast DDS: 'Fast-DDS/src/cpp/utils/threading.hpp:98:57: Error: Parameter pack could not be expanded within '...': set_name_to_current_thread(name, args ...);'. I tried changing the compiler but the problem persists. However, it installed successfully within a container. What could be the reason for this?"Thanks very much.

**Comment by elianalf** [comment]
Hi @MengWWWWW, thanks for using Fast DDS.
According with the details in the [reported issue](https://github.com/eProsima/Fast-DDS/issues/4667), it's possible that there might be a problem with the syntax or usage of the variadic arguments in GCC versions in CentOS 7. Unfortunately, we currently do not support CentOS 7. Which is your current version of GCC? You might want to have a look to [all the versions and OS](https://fast-dds.docs.eprosima.com/en/latest/notes/versions.html#build-system-dependencies) we currently support. If you may find any solutions and you want to contribute to it, we will be happy to review and add this contribution.
